website/frontend/templates/docs/linux.html

```diff
@@ -11,6 +11,22 @@
         <a href="/downloads/#source" title="Source code for MusicBrainz Picard">here</a>.
       </p>
 
+      <h1 id="flatpak">Flatpak</h1>
+      <span>
+        <p>
+          Picard is available on <a href="https://flathub.org">Flathub</a>. This version should work on all modern
+          Linux distributions.</p>
+        </p>
+        <p>
+          First enable the Flathub repository:
+        </p>
+        <pre>flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo</pre>
```

Comment:
I get `flatpak: not found` when doing this. ;) Maybe add something like "Check your distributions documentation for how to install flatpak itself." to the initial paragraph?